README.md

```diff
@@ -60,7 +60,18 @@ negotiation and thereby give rise to bitcoin's first truly risk less interest ra
 $ cargo update
 $ cargo build --release
 ```
-# Run
+
+# Running On Local Regtest Network (Recommended for developers)
+1. Be sure to have a somewhat recent version of bitcoin core installed. Importantly, it must be a version which serves BIP 157 blockfilters.
```

Comment:
If you want to include a link, [this is the BIP 157 blockfilters branch](https://github.com/jimpo/bitcoin/tree/bip157-net) associated with the BIP.

Reply:
Links included. Thanks.